src/types/model.ts

```diff
@@ -400,6 +400,7 @@ export interface DataAccessRequest {
   dmi: DataManagementIncident;
   researchPlans: string;
   closeoutSupplement: Closeout;
+  data: object;
```

Comment:
The meat of a DAR is in `data`. I've added it so we can spread it onto the front end object we expect to hold the initial state.

We may have to do more now because we're fully populating this object now.  **That means the second progress report will by default, have values populated from the first progress report.**. If we need to work around that, we'll have to more completely build out this model to match what the return object actually is so that we can access the specific properties we want out of data when we need them.